README.md

```diff
@@ -0,0 +1,63 @@
+# IronSSH - End-to-end secure file transfer
```

Comment:
We should put our logo above this. And actually into the github project also.  This guy: https://ironcore-website-beta.herokuapp.com/products/ironsftp.  I can do that later after the new website is released.